parm/use_cases/model_applications/medium_range/TCStat_SeriesAnalysis_fcstGFS_obsGFS_FeatureRelative_SeriesByInit.conf

```diff
@@ -106,15 +146,12 @@ TC_STAT_BASIN =
 TC_STAT_CYCLONE =
 TC_STAT_STORM_NAME =
 
-# Stratify by init times via a comma-separate list of init times to
-# include or exclude.  Time format defined as YYYYMMDD_HH or YYYYMMDD_HHmmss
 TC_STAT_INIT_BEG =
```

Comment:
Many of the TC_STAT options are empty here as well; seems to be common for the feature relative use cases. Should these be removed for ease of reading, or are they popular/integral to users and should stay?